Alloy/CodableConformance.swift

```diff
@@ -11,67 +11,3 @@ extension MTLCPUCacheMode: Codable {}
 extension MTLTextureUsage: Codable {}
 extension MTLTextureType: Codable {}
 extension MTLPixelFormat: Codable {}
-
```

Comment:
why you moved it from here?

Reply:
I thought maybe we should keep types extensions in one place

Reply:
then I think it is reasonable to kill `CodableConformance.swift` altogether, because it is very misleading. but I think we can just revert for now and merge new extensions